spec/helpers/webmock_helper.rb

```diff
@@ -0,0 +1,41 @@
+require "webmock/rspec"
+
+module WebMockHelper
+  def mock_json(method, endpoint, response_file, options = {})
+    stub_request(method, endpoint).with(
+      request_for(method, options)
+    ).to_return(
+      response_for(response_file, options)
+    )
+    result = yield
+    expect(a_request(method, endpoint).with(
+             request_for(method, options)
+    )).to have_been_made.once
+    result
+  end
+
+  private
+
+  def request_for(method, options = {})
+    request = {}
+    case method
+    when :post, :put
+      request[:body] = options[:params]
+    else
+      request[:query] = options[:params]
+    end
+    request[:headers] = options[:request_header] if options[:request_header]
+    request
+  end
+
+  def response_for(response_file, options = {})
+    response = {}
+    response[:body] = File.new(File.join(File.dirname(__FILE__), "../fixtures",
+      "#{response_file}.#{options[:format] || :json}"))
+    response[:status] = options[:status] if options[:status]
+    response
+  end
+end
+
+include WebMockHelper
+WebMock.disable_net_connect!
```

Comment:
Please correct me if I'm wrong, but could we completely remove this and use `VCR` instead?